schemas/interface.yml

```diff
@@ -0,0 +1,38 @@
+- name: interface
+  title: Interface
+  group: 2
+  short: Fields to describe observer interface information.
+  description: >
+    The interface fields describe information about ingress and egress interfaces as reported 
+    by an observer.
+
+  reusable:
+    top_level: false
+    expected:
+      - observer.ingress
+      - observer.egress
+
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      example: 10
+      description: >
+        Interface ID as reported by the system (typically SNMP interface ID).
+
+    - name: name
+      level: extended
+      type: keyword
+      example: eth0
+      description: >
+        Interface name as reported by the system. 
+
+    - name: alias
+      level: extended
+      type: keyword
+      example: outside
+      description: >
```

Comment:
Please add a short description to this one

Reply:
done